packages/preview-middleware-client/src/messagebundle.properties

```diff
@@ -37,9 +37,13 @@ ADP_ADD_SUB_PAGE_DIALOG_PAGE_TYPE_LABEL = Page Type
 ADP_ADD_SUB_PAGE_DIALOG_NAVIGATION_LABEL = Navigation
 ADP_ADD_FRAGMENT_NOTIFICATION = Note: The `{0}.fragment.xml` fragment will be created once you save the change.
 ADP_ADD_FRAGMENT_WITH_TEMPLATE_NOTIFICATION = Note: The `{0}.fragment.xml` fragment will be created once you save the changes.
+ADP_CONTROLLER_EXTENSION_EXISTS = An existing controller extension has been found. Please use the previously created extension controller
```

Comment:
A dot at the end of the sentence is missing as I see the ones below have it at the end.